lib/dal/src/diagram/diagram_object.rs

```diff
@@ -0,0 +1,106 @@
+use crate::{
+    diagram::{view::ViewId, DiagramError, DiagramResult},
+    implement_add_edge_to,
+    workspace_snapshot::node_weight::{
+        category_node_weight::CategoryNodeKind,
+        diagram_object_node_weight::{DiagramObjectKind, DiagramObjectNodeWeight},
+        traits::SiVersionedNodeWeight,
+        NodeWeight,
+    },
+    DalContext, EdgeWeightKind, EdgeWeightKindDiscriminants,
+};
+use jwt_simple::prelude::{Deserialize, Serialize};
```

Comment:
This is probably not the right place to be importing Deserialize, Serialize (should come from serde)

Reply:
good call, fixed